.gitignore

```diff
@@ -18,4 +18,4 @@ project/plugins/project/
 .scala_dependencies
 .worksheet
 .idea
-*.iml
\ No newline at end of file
+.scalafmt.conf
```

Comment:
I'm confused that .scalafmt.conf is included in the branch.
maybe you want to delete the git cache?